internal/alias/config.go

```diff
@@ -0,0 +1,129 @@
+package alias
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	scwAliasConfigPath = "SCW_ALIAS_CONFIG_PATH"
```

Comment:
I would prefer to have a `SCW_CLI_CONFIG_PATH` that we can reuse for multiple configuration subjects